docs/wiring/blower.rst

```diff
@@ -0,0 +1,43 @@
+.. _Blower Harness:
+
+Blower Harness
+======================
+
+.. csv-table:: **BOM**
+   :widths: 5, 60, 5, 5, 5, 10, 10, 10
+   :header-rows: 1
+   :delim: 0009
+   :file: ./blower.bom.tsv
+
+.. image:: blower.png
+  :width: 800
+  :alt: Wiring diagram
+
+You will need all the tools listed.
+
+* Using items from above BOM as well as:
+   * 3mm heat shrink tubing [:ref:`ew-shrink3`]
+   * 5mm heat shrink tubing [:ref:`ew-shrink5`]
+   * Heat gun
+   * Molex Micro-Fit 3.0 Crimper
+   * Wire Strippers
+   * Soldering iron
+   * Solder
+* Instructions:
+   #. Cut 3 pieces of the 5mm heat shrink, 15mm long each.
+   #. Thread the 5mm heat shrink onto the the entire bundle of 4 wires on the blower.  Do not shrink the heat shrink yet.
+   #. Insert the 2x pre-crimped Mega-Fit wires into the Megafit 2x2 housing.
+   #. Do a gentle pull test to confirm proper seating of the contacts.
+   #. Cut the cable at a length of 50mm from the connector.
+   #. Strip 5mm of insulation from the end of the pre-crimped wires and from the ends of the red and black pigtails of the blower.
+   #. Cut 2 pieces of 3mm heat shrink, 15mm long.
+   #. Slide the 2 pieces of heat shrink onto the red and black wires of the blower.
+   #. Solder the red and black wires to the Mega-Fit connector cables according to the diagram (pin 1 on the Mega-Fit connector is indicated by a stripe on the connector body).
+   #. Slide the heat shrink over the soldered portion and use the heat gun to shrink them in place.
+   #. Strip the insulation from the blue pigtail of the blower, using the Micro-Fit crimp contact as a guide for the legth of insulation to strip.
+   #. Using the micro-fit crimper, crimp the contact onto the end of the blue wire.
+   #. Insert the contact and wire into the micro-fit connector housing according to the diagram.
+   #. Do a gentle pull test to confirm proper seating of the contacts.
+   #. Distribute the 5mm heat shrink tubing evenly along the harness and shrink them with the heat gun.
+
```

Comment:
I would say something about what to do with the yellow wire? I cut the exposed contact off and tucked it into one of the omnibus shrink tube sections to seal it away from accidentally contacting something.

Reply:
done.